frontend/no.4/common-component-app/stories/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import { VFC } from 'react';
+import './button.css';
+
+interface Props {
+  children: string;
+  color?: 'red' | 'blue' | 'green';
+  size?: 'small' | 'medium' | 'large';
+  disabled: boolean;
+  onClick?: () => void;
```

Comment:
```suggestion
  onClick?: React.ChangeEventHandler<HtmlButtonElement>;
```

個人の好みかもしれませんが、汎用的なボタンなら標準の `<button>` と同じ型にしたいな〜と思いました :eyes: